pkg/loadbalancer/module.go

```diff
@@ -136,6 +137,58 @@ func (h *HaproxyModule) Init() {
 	}
 }
 
+type KubevipModule struct {
+	common.KubeModule
+	Skip bool
+}
+
+func (k *KubevipModule) IsSkip() bool {
+	return k.Skip
+}
+
+func (k *KubevipModule) Init() {
+	k.Name = "InternalLoadbalancerModule"
+	k.Desc = "Install internal load balancer"
+
+	getInterface := &task.RemoteTask{
+		Name:     "GetNodeInterface",
+		Desc:     "Get Node Interface",
+		Hosts:    k.Runtime.GetHostsByRole(common.Master),
+		Action:   new(GetInterfaceName),
+		Parallel: true,
+	}
+
+	kubevipManifestOnlyFirstMaster := &task.RemoteTask{
+		Name:     "GenerateKubevipManifest",
+		Desc:     "Generate kubevip manifest at first master",
+		Hosts:    k.Runtime.GetHostsByRole(common.Master),
+		Prepare:  new(common.OnlyFirstMaster),
+		Action:   new(GenerateKubevipManifest),
+		Parallel: true,
+	}
+
+	kubevipManifestNotFirstMaster := &task.RemoteTask{
+		Name:     "GenerateKubevipManifest",
+		Desc:     "Generate kubevip manifest at other master",
+		Hosts:    k.Runtime.GetHostsByRole(common.Master),
+		Prepare:  new(common.NotFirstMaster),
+		Action:   new(GenerateKubevipManifest),
+		Parallel: true,
+	}
+
+	if exist, _ := k.BaseModule.PipelineCache.Get(common.ClusterExist); exist == true {
```

Comment:
Get() return an interface{}, this logic looks a bit confusing, so I prefer to use a more clear way:
```
if exist, _ := k. PipelineCache.GetMustBool(common.ClusterExist); exist {
}
```